appservice/src/deploy/deployZip.ts

```diff
@@ -44,6 +44,16 @@ export async function deployZip(client: SiteClient, fsPath: string, configuratio
             const kuduClient: KuduClient = await getKuduClient(client);
             await kuduClient.pushDeployment.zipPushDeploy(fs.createReadStream(zipFilePath), { isAsync: true });
             await waitForDeploymentToComplete(client, kuduClient);
+            if (asp && asp.sku && asp.sku.tier && asp.sku.tier.toLowerCase() === 'basic') {
```

Comment:
I thought Matt said we only needed to do this on Linux - not windows?

Also I don't think we need this on Function Apps?